src/renderer/shared/api/xcm/xcmService.ts

```diff
@@ -104,8 +126,22 @@ const JunctionType: Record<string, string> = {
   accountId: 'AccountId32',
   generalIndex: 'GeneralIndex',
 };
+
 type JunctionTypeKey = keyof typeof JunctionType;
 
+const JunctionWeight: Record<JunctionTypeKey, number> = {
```

Comment:
I suggest to rename it to something like `JunctionHierarhyLevel` to not consude it with weight calculations